www/archive.inc

```diff
@@ -17,6 +17,9 @@ function ArchiveTest($id, $store_result = true, $delete = false) {
     if ( strpos($id, '.') === false &&                           // make sure it isn't a relay test
         (array_key_exists('archive_dir', $settings) ||           // local mount
         array_key_exists('archive_s3_server', $settings)) ) {    // S3 Interface (Internet Archive or standard)
+        if ($settings['archive_dir'] == 'none') { // don't archive anything

```

Comment:
Can you make this a case-insensitive comparison just to be safe (and potentially also allow for /dev/null)?

Something like:
```
if (strcasecmp($settings['archive_dir'], 'none') === false || strcasecmp($settings['archive_dir'], '/dev/null') === false) {
```